erpnext/agriculture/doctype/harvest/harvest.py

```diff
@@ -3,8 +3,21 @@
 # For license information, please see license.txt
 
 from __future__ import unicode_literals
-# import frappe
+import frappe
 from frappe.model.document import Document
+from bloomstack_core.bloomtrace import make_integration_request
 
 class Harvest(Document):
 	pass
+
+@frappe.whitelist()
+def finish_harvest(harvest):
+	frappe.db.set_value("Harvest", harvest, "is_finished", 1)
+	make_integration_request("Harvest", harvest)
```

Comment:
```suggestion
	make_integration_request("Harvest", harvest)
```
use hooks to make integration request
do not import bloomstack_core in erpnext